plugins/optimization-detective/class-od-html-tag-processor.php

```diff
@@ -497,15 +497,27 @@ public function release_bookmark( $name ): bool {
 	 * A breadcrumb consists of a tag name and its sibling index.
 	 *
 	 * @since 0.4.0
+	 * @since n.e.x.t Renamed from get_breadcrumbs() to get_indexed_breadcrumbs().
 	 *
 	 * @return Generator<array{string, int}> Breadcrumb.
 	 */
-	private function get_breadcrumbs(): Generator {
+	private function get_indexed_breadcrumbs(): Generator {
 		foreach ( $this->open_stack_tags as $i => $breadcrumb_tag_name ) {
 			yield array( $breadcrumb_tag_name, $this->open_stack_indices[ $i ] );
 		}
 	}
 
+	/**
+	 * Gets breadcrumbs for the current open tag.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string[] Breadcrumb.
+	 */
+	public function get_breadcrumbs(): array {
+		return $this->open_stack_tags;
+	}
```

Comment:
Let's try to align this as closely as possible with [`WP_HTML_Processor::get_breadcrumbs()`](https://github.com/WordPress/wordpress-develop/blob/daad8631f707f730530711ed874efe94c7fec968/src/wp-includes/html-api/class-wp-html-processor.php#L1138-L1160), so like:

```suggestion
	/**
	 * Computes the HTML breadcrumbs for the currently-matched node, if matched.
	 *
	 * Breadcrumbs start at the outermost parent and descend toward the matched element.
	 * They always include the entire path from the root HTML node to the matched element.
	 *
	 * @since n.e.x.t
	 *
	 * @return string[] Array of tag names representing path to matched node.
	 */
	public function get_breadcrumbs(): array {
		return $this->open_stack_tags;
	}
```

Note that it seems like `WP_HTML_Processor::get_breadcrumbs()` doesn't ever actually return `null`. I've [inquired](https://wordpress.slack.com/archives/C05NFB818PQ/p1733162734566649) about that.